### PR TITLE
DOC: physics/units/examples.rst: wording, punctuation

### PR DESCRIPTION
**Pull request by westurner**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: doc/src/modules/physics/units/examples.rst [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
 
 Dimensional analysis
 ====================
 
-We will start from the second Newton's law
+We will start from Newton's second law
 
 .. math::
-    m a = F
+    m * a = F
```
[the remark this answers is not in the document]
Comment (westurner) [reply doc/src/modules/physics/units/examples.rst]: I've removed the extra ``*``.

It'd be great if GitHub would render ``.. math:`` directives.

[the remark this answers is not in the document]
Comment (moorepants) [reply doc/src/modules/physics/units/examples.rst]: It used to! :)

[the remark this answers is not in the document]
Comment (westurner) [reply doc/src/modules/physics/units/examples.rst]: https://github.com/github/markup/issues/897
- GitLab supports MathJax client-side rendering.
- GitHub used to.
- It's certainly understandable that server-side LaTeX is not securable; but client-side should be a relatively trivial fix: grep for a ``.. math::`` directive error and serve a different JS bundle when MathJax is needed.

**Comment by westurner** [comment]
Thanks!

**Pull request merged**